index.js

```diff
@@ -10,3 +10,39 @@ mongoose.connect('mongodb://localhost/recipeApp', { useNewUrlParser: true })
     console.error('Error connecting to mongo', err);
   });
 
+  Recipe.create({
+    title: 'Hot Pot',
+    level: 'Easy Peasy',
+    ingredients: 'tofu',
+    cuisine: 'Asian',
+    dishType: 'Dish',
+    image:'https://bmexdi064h-flywheel.netdna-ssl.com/wp-content/uploads/2018/02/Asian-Red-Curry-Hot-Pot-foodiecrush.com-047.jpg',
+    durantion: '20',
+    creator: 'Yang',
+    created: '01/01/1200',
+
+  })
+
```

Comment:
you can add a promise after your create method so you can have feedback about the creation ! it is not mandatory but it can help you

`  .then(recipeDoc => {
    console.log("New recipe CREATED correctly ! Yummmm.", recipeDoc);
  })
  .catch(err => {
    console.log("Recipe's creation failed...", err);
  });`